src/gui/src/views/users/settings/UserSettings.vue

```diff
@@ -0,0 +1,175 @@
+<template>
```

Comment:
Functionality not given 
`UserSettings.vue:69 [intlify] Not found 'settings.open_item_source' key in 'en' locale messages.`

Reply:
see https://github.com/ait-cs-IaaS/Taranis-NG/issues/99

Reply:
Language selection works perfectly, however after selecting a language the "save" button throws an error --> 

`marshmallow.exceptions.ValidationError: {'language': ['Unknown field.']}`

@b3n4kh  Maybe we can update this in the #99 issue

Reply:
Should Hotkeys be selectable?  current behaviour: `Not found 'settings.open_item_source' key in 'de' locale messages.`

is '@/views/users/settings/UserWordLists.vue' not found realated?